README.md

```diff
@@ -112,6 +112,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Mattias Petter Johansson](#mattias-petter-johansson) - **streaming:** JavaScript, Node.js, Programming in general
 - [MechJack](#mechjack) - **streaming:** Java, JavaScript, React, Framework development, Programming in general
 - [Micah Elizabeth Scott](#micah-elizabeth-scott-scanlime) - **streaming:** Reverse Engineering, Rust, C++, Hardware
+- [Michael Henderson](#michael-henderson) - **streaming:** Fullstack development
```

Comment:
The anchor tag (`#michael-henderson`) isn't correct. Because of all the aka stuff that was added, GitHub generates `#michael-henderson-aka-robedcoder--aka-leadcoder` as the proper anchor tag.

Reply:
Good to know. I will close this PR and re-submit. I need to apply that fix to my first PR from a few weeks ago.